performjoin.go

```diff
@@ -213,6 +237,40 @@ func PerformJoin(
 		}
 	}
 
+	// get the membership event of the room creator, so we can store the mxid_mapping
+	// TODO: better way?
+	if roomVersion == RoomVersionPseudoIDs {
```

Comment:
interesting... should we be doing a similar thing with the respSendJoin state events to add mappings for all the currently joined users? (not just the creator)
seems like an expensive operation, but possibly necessary...

Reply:
Good point, definitely makes sense to add that.